src/pages/DotaAPI/FetchFunction.jsx

```diff
@@ -0,0 +1,7 @@
+const getAPI = async arg => {
+    let response = await fetch(arg);
```

Comment:
will merge, but i would say https://api.opendota.com/api should be include here

Reply:
noted

Reply:
In this case, will be 

let response = await fetch(`https://api.opendota.com/api${arg}`);

?

Reply:
```
const API_HOST = https://api.opendota.com/api

const callApi = async (endpoint){
  let response = await fetch(`API_HOST/${endpoint}`);
}
```